tests/NLog.UnitTests/Config/InternalLoggingTests.cs

```diff
@@ -44,7 +44,7 @@ public void InternalLoggingConfigTest1()
             using (var scope = new InternalLoggerScope())
             {
                 CreateConfigurationFromString(@"
-<nlog internalLogFile='c:\file.txt' internalLogLevel='Trace' internalLogToConsole='true' internalLogToConsoleError='true' globalThreshold='Warn' throwExceptions='true'>
+<nlog internalLogFile='c:\file.txt' internalLogLevel='Trace' internalLogToConsole='true' internalLogToConsoleError='true' globalThreshold='Warn' throwExceptions='true' internalWriteToDiagnostics='true'>
```

Comment:
I'm aware that the current tests are not that good. Still I like to ask to write some more tests as we need more tests in the end (code coverage of 73-74% is not that good)

You can use https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTraceListenerTests.cs#L64-L307 to check how-to read the trace.